baitap-19-10-2020/bai-1.js

```diff
@@ -0,0 +1,17 @@
+const salaryCalculation = (salary) => {

```

Comment:
```suggestion
const calcSalary = salary => {
```
Hoặc
```suggestion
const getSalary = salary => {
```

Nếu hàm chỉ có 1 param thì có thể bỏ dấu `()` đi